cumulus/tasks/sync-granule/tests/sync_granule_test.js

```diff
@@ -98,6 +100,68 @@ test('download Granule from HTTP endpoint', (t) => {
     });
 });
 
+test('download Granule from S3 endpoint', (t) => {
```

Comment:
Consider updating this test to use async/await.  It would be easier to read and less error prone.